main.go

```diff
@@ -81,16 +92,34 @@ func main() {
 
 	setupLog := log.Log
 
-	var metricsAddr string
-	var enableLeaderElection bool
-	var probeAddr string
+	var (
+		metricsAddr          string
+		enableLeaderElection bool
+		probeAddr            string
+		configFile           string
+		err                  error
+	)
+
 	flag.StringVar(&metricsAddr, "metrics-bind-address", ":8080", "The address the metric endpoint binds to.")
 	flag.StringVar(&probeAddr, "health-probe-bind-address", ":8081", "The address the probe endpoint binds to.")
 	flag.BoolVar(&enableLeaderElection, "leader-elect", false,
 		"Enable leader election for controller manager. "+
 			"Enabling this will ensure there is only one active controller manager.")
+	flag.StringVar(&configFile, "config", "",
+		"The operator will load its initial configuration from this file. "+
+			"Omit this flag to use the default configuration values. "+
+			"Command-line flags override configuration from this file.")
 	flag.Parse()
 
+	options := ctrl.Options{Scheme: scheme}
```

Comment:
`options` is being used? 

This is a mix of config file driven  configuration with cli params. I think we should pick one of the options. I tend to prefer the config file driven configuration, but I would also accept the other approach.

Reply:
Indeed, `options` were not applied. Now they are after https://github.com/Kuadrant/kuadrant-operator/pull/54/commits/6dc60a2d0de69a6422f0d9b6af10c7e76a09b9ac.

I don't have any particular preference, maybe we could discuss it with the rest and refactor accordingly? WDYT?